mongoDbTest.js

```diff
@@ -226,7 +226,19 @@ app.post("/login", async (req, res) => {
 	}
 })
 
-
+app.post("/logout", (req, res) => {
+	// TODO: Not sure how to do the error handling here
+	if(req.session) {
+		req.session.destroy(err => {
+			if(err) {
+				// TODO: Should this be a 400 or 500?
+				res.status(400).send("Unable to logout");
```

Comment:
`400` means the server thinks it was sent a malformed request – this is not the case here.

> The server cannot or will not process the request due to something that is perceived to be a client error (e.g., malformed request syntax, invalid request message framing, or deceptive request routing).

Some StackOverflow answers indicate that `422` could be the right error code here. (Maybe; if the error is due to there not being a session to log out of.)

> The 422 (Unprocessable Entity) status code means the server understands the content type of the request entity (hence a 415(Unsupported Media Type) status code is inappropriate), and the syntax of the request entity is correct (thus a 400 (Bad Request) status code is inappropriate) but was unable to process the contained instructions.

That said, I'm not 100% certain when these error cases would be hit. So, hm.

Reply:
```suggestion
				res.status(422).send("Unable to logout");
```